ui/k-toolbar/main.kytos

```diff
@@ -168,6 +180,39 @@ module.exports = {
       });
 
     },
+    async fetch_dpids() {
```

Comment:
Could you rename this function as `fetch_interfaces`? Primarily on `GET /v3/interfaces` you're fetching interfaces and not dpids. You have the dpid (switch) value on each interface though but that's not a major relevant thing in this context. 

Great documentation on this method too, if you could also update it, that'd be great.